test/langtools/tools/javac/6341866/T6341866.java

```diff
@@ -67,7 +67,7 @@ enum ImplicitType {
 
     enum AnnoType {
         NONE,           // no annotation processing
-        SERVICE,        // implicit annotation processing, via ServiceLoader
+     // SERVICE,        // implicit annotation processing, via ServiceLoader
```

Comment:
why keeping this if it won't be used?

Reply:
> why keeping this if it won't be used?

Sure; I can delete that before the changeset goes back.